docs/notebooks/PCMSolvent.ipynb

```diff
@@ -1,36 +1,122 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        print(f"iteration: {i} \t norm: {V_R.norm():.5f} \t update: {update:.5e} \t energy: {E_r:.7f} \t energy update: {dE_r:.5e}") 
</pre><p>I suggest you truncate the representation of these floating-point numbers, so that stuff fits on line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/vampyr/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>